hack/tracing-setup-tempo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install tempo grafana/tempo --namespace tempo --create-namespace
```

Comment:
I'm curious on your thoughts of installing with the [kustomize manifests](https://github.com/grafana/tempo-operator/tree/main/config) instead of helm similar to our other dependencies and having a new make target like `make deploy-tempo`